src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/Processor.java

```diff
@@ -127,10 +127,7 @@ private Boolean deleteBlob(
         try {
             // Lease needs to be broken before deleting the blob. 0 implies lease is broken immediately
             cloudBlockBlob.breakLease(0);
-            boolean deleted = cloudBlockBlob.deleteIfExists();
-            if (!deleted) {
-                handleBlobDeletionError(envelope, null);
```

Comment:
This would actually fail if it was called, because of NullPointerException. #378 logs a warning instead.

Reply:
"love" how `boolean` can be `null`

Reply:
Primitive boolean can't, but `handleBlobDeletion` passes null where exception is expected.